### ISSUE TITLE
i can't wear it.

### ISSUE THREAD

**Issue by JamRobLam** [Darkona/AdventureBackpack2#42]
I downloaded the mod and I'm certain i installed it correctly being i can craft the backpack. my problem is though i can't equip the backpack, nor can i access the inventory by right clicking with it in hand.

**Comment by Brayferd** [comment]
Have same problem

**Comment by Darkona** [comment]
This is VERY strange. Can I see a list of other mods you guys have?

**Comment by JamRobLam** [comment]
I think I figured it out though I'm not sure if this is your problem Brayferd. i really like this mod a lot and wanted it to work so i re downloaded it and switched over to the new file, and then tried it worked. come to find out though if your looking at a block even if its right outside your range it will try to place to backpack. Try looking at something really far or in the sky. How ever i had some problems with taking the backpack from the "TooManyItems" NEI and i could not craft with them. but for reference i was using:
Too many items. - Marglyph
CraftGuide. - Uristqwerty
Minions mod. - AtomicStryker
Balkon’s WeaponMod. - BalkondeurAlpha

I did uninstall TMI and everything is working wonderfully. its an awesome mod.
